WebhookNotification/handler.ts

```diff
@@ -251,12 +260,26 @@ export const getWebhookNotificationHandler = (
           const webhookNotification =
             errorOrWebhookNotification.right.channels.WEBHOOK;
 
+          const userProfile = await pipe(
+            userProfileReader({
+              fiscalCode: message.fiscalCode
+            }),
+            // We are using T.map(E.getOrElseW(..)) because
+            // TE.getOrElseW is unable to infer `userProfile` type
```

Comment:
Is this an fp-ts bug?

Reply:
My bad. With `TE.getOrElse` it works fine.  
Done in [[#IOCIT-141] fix TE.getOrElse return type on userProfile](https://github.com/pagopa/io-functions-services/pull/232/commits/5ded0e720d595ea51386e31d527d04c6a46ae694)